src/app/endround/endround.component.scss

```diff
@@ -0,0 +1,151 @@
+.endround {
+    position: absolute;
+    left: 26%;
+    top: 100%;
+    width: 875px;
+    height: 200px;
+    overflow-x: hidden;
+    animation-name: changepos-2;
+    animation-duration: 3.95s;
+    animation-fill-mode: both;
+}
+
+@keyframes changepos-1 {
+    0%   {top: 40.5%;}
+    90%  {top: 40.5%;}
+    100% {top:150%;}
+}
+
+@keyframes changepos-2 {
+    0%   {top: 100%;}
+    25%  {top: 100%;}
+    90%  {top: 100%;}
+    100% {top:82%;}
+}
+
+.endround-box {
+    position: relative;
+    width: 875px;
+    height: 200px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: stretch;
+    // margin: 10px;
+    // animation-name: slide-in;
+    // animation-duration: 1s;
+    // animation-fill-mode: both;
+    // transition-timing-function: ease-in-out;
+}
+
+@keyframes slide-in {
+    0%   {left: -102%;}
+    75%  {left: -102%;}
+    100% {left: 0%;}
+}
+
+.win-side-image {
+    position: relative;
+    max-height: 120px;
+}
+.win-side-image-box {
+    display: flex;
+    height: 140px;
+    align-items: center;
+    scale: 0.9;
+}
+.win-side {
+    position: relative;
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 30%;
+}
+
+.win-side-text {
+    position: relative;
+    flex-grow: 2;
+    margin-top: 10px;
+    width: fit-content;
+    text-shadow: 0px 1.21px 4.83px rgba(0, 0, 0, 0.12);
+    font-family: "Montserrat", Helvetica;
+    font-weight: 700;
+    color: #ffffff;
+    font-size: 25px;
+    font-style: normal;
+    text-align: center;
+    // letter-spacing: 1.2px;
+    line-height: normal;
+}
+
+.info-box {
+    position: relative;
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: center;
+    width: 50%;
+}
+.info-image {
+    position: relative;
+    max-height: 120px;
+    max-width: 400px;
+}
+.info-image-box{
+    display: flex;
+    height: 130px;
+    align-items: center;
+    scale: 0.9;
+}
+.round-number {
+    position: relative;
+    width: fit-content;
+    text-shadow: 0px 1.21px 4.83px rgba(0, 0, 0, 0.12);
+    font-family: "Montserrat", Helvetica;
+    font-weight: 700;
+    color: #ffffff;
+    font-size: 25px;
+    font-style: normal;
+    text-align: center;
+    // letter-spacing: 1.2px;
+    line-height: normal;
+}
+
+.intro-box {
+    position: absolute;
+    left: 26.23%;
+    top: 40.5%;
+    width: 875px;
+    height: 200px;
+    overflow-x: hidden;
```

Comment:
This should be `overflow: hidden` to prevent a scrollbar showing up sometimes

Reply:
Same for other occurences most likely

Reply:
What browser did you use to see that scrollbar, I'm using Chrome 131 and haven't ever seen that scrollbar tho

Reply:
> What browser did you use to see that scrollbar, I'm using Chrome 131 and haven't ever seen that scrollbar tho

Chromium 131, might be because of 4k monitor and/or using DevTools for forcing it to 1080p

Reply:
could you try it on OBS Browser Source or vMix to see if that scrollbar
still exists ? I see overall performance especially on CSS animations
better.